CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Change log for Microsoft365DSC
 
 # 1.22.727.1
-
+* AADRoleSetting
```

Comment:
Please create an Unreleased section above the 1.22.727.1 and add this entry under there.